app/assets/javascripts/3dbio_viewer/src/webapp/components/molecular-structure/usePluginRef.ts

```diff
@@ -429,6 +443,13 @@ function updateSequenceView(args: {
     }
 }
 
+function pdbeMolstarSequenceEventCompletedWrapper(callback: () => void) {
+    return (sequence: any) => {
```

Comment:
[minor] as discussed in the other PR, when we don't really care about the type, it's safe "unknown"